src/main/java/dev/cdevents/CDEventTypes.java

```diff
@@ -5,97 +5,713 @@
 import java.time.OffsetDateTime;
 import java.util.UUID;
 
+import dev.cdevents.constants.CDEventConstants;
 import io.cloudevents.CloudEvent;
+import io.cloudevents.CloudEventData;
 import io.cloudevents.core.v03.CloudEventBuilder;
+//import org.graalvm.compiler.nodes.calc.ObjectEqualsNode;
 
 public final class CDEventTypes {
 
     private CDEventTypes() {
     }
 
     /**
-     * Creates a continuous delivery Pipeline run events
+     * Creates continuous delivery Pipeline run finished events
      * using {@link CloudEventBuilder}
      * and returns {@link CloudEvent} object.
      *
-     * @param pipelineRunEventType
-     * @param pipelineRunId
-     * @param pipelineRunName
-     * @param pipelineRunStatus
-     * @param pipelineRunURL
-     * @param pipelineRunErrors
-     * @param pipelineRunData
+     * @param eventType
+     * @param id
+     * @param pipelineName
+     * @param url
+     * @param errors
+     * @param outcome

```

Comment:
Use Auto "generate element comment" for each method to add this Java doc and avoid declaring missing params, this resolves many of Lint code errors